js/background.js

```diff
@@ -802,6 +802,18 @@
       appView.openConversation(groupId, {});
     };
 
+    // $(document).ready(() => {
+    //   window.settingsView = new Whisper.SessionSettingsView({
+    //     el: $('#settings-container'),
+    //   });
+    //   window.settingsView.render();
+    // });
+
+    window.generateID = () =>
+      Math.random()
+        .toString(36)
+        .substring(3);
+
```

Comment:
We should either keep this one or the one in ComponentTools.tsx

Reply:
Resolved